lib/logic.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class TicTacLogic
+  attr_accessor :player1, :player2
+  attr_reader :board, :play, :playert
+
+  def initialize
+    @play = true
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    @playert = true
+  end
+
+  def move(num)
+    if @board[num - 1].is_a?(Integer) && num >= 0 && num <= 9
```

Comment:
Usually it recommended to check for index validity before the value into the array at that index position
Not that is not working but it's not a good habit
```ruby
if num >= 0 && num <= 9 && @board[num - 1].is_a?(Integer) 
```
is preferable.

Take the habit to write it like this